### PR TITLE
New plugins blog

### PR DESCRIPTION
**Pull request by kwombach12**

### CODE REVIEW

File: blog/2024-11-21-new-plugins/index.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+slug: Latest Updates: New OpenCost Plugins and 1,000 Incentives for Community Developers
```
Comment (chipzoller) [inline blog/2024-11-21-new-plugins/index.mdx:2]:
```suggestion
slug: "Latest Updates - New OpenCost Plugins and 1,000 Incentives for Community Developers"
```

File: blog/2024-11-21-new-plugins/index.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+slug: "Latest Updates - New OpenCost Plugins and 1,000 Incentives for Community Developers"
```
Comment (chipzoller) [inline blog/2024-11-21-new-plugins/index.mdx:2]:
```suggestion
slug: "Latest Updates - New OpenCost Plugins and $1,000 Incentives for Community Developers"
```

Try adding dollar sign back in.

Comment (kwombach12) [reply blog/2024-11-21-new-plugins/index.mdx]: re-adding the dollar sign worked.

**Review by joe-terrible — APPROVED** [review]
Looks great!

**Pull request merged**